packages/scm/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@theia/scm",
+  "version": "0.6.0",
+  "description": "Theia - Source control Extension",
+  "dependencies": {
+    "@theia/core": "^0.6.0",
+    "@theia/editor": "^0.6.0",
+    "@theia/filesystem": "^0.6.0",
+    "@theia/plugin-ext": "^0.6.0"
```

Comment:
there are some dependencies missing here. I saw `@theia/navigator`, `diff` and `ts-md5`

Reply:
fixed